hedera-node/src/test/java/com/hedera/services/txns/validation/ContextOptionValidatorTest.java

```diff
@@ -604,8 +607,17 @@ void memoCheckWorks() {
 	}
 
 	@Test
-	void rejectsInvalidBatchSize(){
+	void rejectsInvalidBatchSize() {
 		var list = LongStream.range(0, 1000).boxed().collect(Collectors.toList());
 		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeBurnCheck(list.size()));
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeMintCheck(list.size()));
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxBatchSizeWipeCheck(list.size()));
+		assertEquals(BATCH_SIZE_LIMIT_EXCEEDED, subject.maxNftTransfersLenCheck(list.size()));
+	}
+
+	@Test
+	void rejectsInvalidMetadata() {
+		assertEquals(QUERY_RANGE_LIMIT_EXCEEDED, subject.nftMaxQueryRangeCheck(0, 1001));
+		assertEquals(METADATA_TOO_LONG, subject.nftMetadataCheck(ByteString.copyFromUtf8("aaaaaaaaaaaaaaa").toByteArray()));
 	}
```

Comment:
You must mock the `dynamicProperties`. Atm they are `0`. We have to use some sensible number